restserver/src/main/java/org/stroganov/rs/controller/AuthorController.java

```diff
@@ -0,0 +1,57 @@
+package org.stroganov.rs.controller;
+
+import org.stroganov.entities.Author;
+import org.stroganov.rs.filter.JWTTokenNeeded;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+
+@Path("/api/author")
+public class AuthorController extends Controller {
+
+    @POST
+    @Path("/")
+    @Consumes({MediaType.APPLICATION_JSON})
+    @Produces({MediaType.APPLICATION_JSON})
+    @JWTTokenNeeded
+    public Response addAuthor(Author author) throws IOException {
+        boolean operationResult = libraryDAO.addAuthor(author);
+        logger.info("Result of addAuthor: " + operationResult);
+        return Response.status(200)
+                .entity("true")
+                .build();
+    }
+
+    @GET
+    @Path("/{authorName}")
+    @Consumes({MediaType.APPLICATION_JSON})
+    @Produces({MediaType.APPLICATION_JSON})
+    @JWTTokenNeeded
+    public Response findAuthor(@PathParam("authorName") String authorName) {
+        Author author = libraryDAO.findAuthor(authorName);
+        return Response.status(200)
+                .entity(author)
+                .build();
+    }
+
+    @DELETE
+    @Path("/{authorName}")
+    @Consumes({MediaType.APPLICATION_JSON})
+    @Produces({MediaType.APPLICATION_JSON})
+    @JWTTokenNeeded
+    public Response deleteAuthorWithAllHisBooks(@PathParam("authorName") String authorName) throws IOException {
+        Author author = libraryDAO.findAuthor(authorName);
+        if (author == null) {
+            return Response.status(404)
+                    .entity(authorName + " not found")
+                    .build();
+        }
+        boolean operationResult = libraryDAO.deleteAuthorWithAllHisBooks(author);
+        logger.info("Result of deleteAuthorWithAllHisBooks: " + operationResult);
+        return Response.status(200)
+                .entity(operationResult)
```

Comment:
```suggestion
                .entity("Delete "+ operationResult ? " successfully" : " not successfully") // meaningful responce
```

Reply:
But in client i suggest true ore false ...